doc/cookbook/source/examples/neural_nets/feedforward_net_regression.rst

```diff
@@ -0,0 +1,45 @@
+==================================
+Feedforward Network for Regression
+==================================
+
+This page illustrates the usage of feedforward networks for regression. For more details about feedforward networks, `see this feedforward network cookbook <http://shogun.ml/cookbook/latest/examples/neural_nets/feedforward_net_classification.html>`_.
```

Comment:
pls use the "doc" style of referencing.
Also pls do that for all other cookbooks you did (see my email, separate patch)